### PR TITLE
Feature/services tabs

### PR DESCRIPTION
**Pull request by JFozzie**
Services Tabs component

Created files:

- _servicesTab.js
- _servicesTab.scss
- serviceComponent.hbs
- servicesTab.hbs

Trello ticket
https://trello.com/c/HNznSYPh

Slug:
http://localhost:8080/servicestabs.html

https://user-images.githubusercontent.com/111401432/202648293-2c07b2d4-f3f2-4721-a3e3-e86f34ae7578.mp4

### CODE REVIEW

File: templates/servicesTab.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+  <div class="tab__name">
```
Comment (jrmahechad) [inline templates/servicesTab.hbs]: @JFozzie Can you update this using a button tag? This will improve accessibility.

**Review by jrmahechad — CHANGES_REQUESTED** [review]
@JFozzie great job!. Pnly a small comment to improve accessibility

File: templates/serviceComponent.hbs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class="service-tabs__wrapper">
+  <div class="service-tabs">
+    <div class="service-tabs__tab--n">
+      {{#each serviceTabs}}
```
Comment (jrmahechad) [inline templates/serviceComponent.hbs]: @jsebax Do you know if we can call a diferent hbs file here?  Looks like we need to split this in two components one serviceTabsHeader and other serviceTabsContent.  

Jordy is duplicating the content and then removing it from JS

Comment (jsebax) [reply templates/serviceComponent.hbs]: Ok. It sounds good. Let's split the content into two components

**Comment by jsebax** [comment]
@JFozzie You are missing the Component title that has the shape. You can implement the different shape variations since they are now included in the Figma design. Please review
image

**Review by jrmahechad — APPROVED** [review]

**Review by jsebax — APPROVED** [review]
Good job 👍

**Pull request merged**